dmoj/graders/custom.py

```diff
@@ -1,13 +1,21 @@
-import os
+from pathlib import Path
 
+from dmoj.error import InternalError
 from dmoj.judgeenv import get_problem_root
 from dmoj.utils.module import load_module_from_file
 
 
 class CustomGrader:
     def __init__(self, judge, problem, language, source):
         self.judge = judge
-        self.mod = load_module_from_file(os.path.join(get_problem_root(problem.id), problem.config['custom_judge']))
+        custom_judge = problem.config['custom_judge']
+        filename = Path(get_problem_root(problem.id), custom_judge)
+        if not filename.exists():
+            filename = Path(__file__).parent / custom_judge
```

Comment:
What's the rationale behind making everything importable via `CustomGrader`?

Reply:
This was @Xyene's suggestion when I asked him how to implement the GCJ grader

Reply:
Yeah but now `standard`, `interactive`, `signature_grader` can be used in two different ways. Not sure if I want that.

Reply:
I'm not sure what @Xyene thinks, but one solution I have is to rename the `Grader` classes back to what they originally were so they can't be imported (because the custom grader requires a `Grader` class specifically)